ansible_events_ui/db/migrations/versions/202209061903_d5f5e14c698a_add_audit_rules_table.py

```diff
@@ -0,0 +1,71 @@
+"""Add audit rules table
+
+Revision ID: d5f5e14c698a
+Revises: c1eee0e47fc1
+Create Date: 2022-09-06 19:03:44.658149+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "d5f5e14c698a"
+down_revision = "c1eee0e47fc1"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Please remove these comments once you manually verified the correctness of the migration.